lib/omnibus/packagers/msi.rb

```diff
@@ -47,48 +47,26 @@ class Packager::MSI < Packager::Base
       FileSyncer.glob("#{resources_path}/assets/*").each do |file|
         copy_file(file, "#{resources_dir}/assets/#{File.basename(file)}")
       end
+
+      copy_file(resource_path('CustomActionFastMsi.CA.dll'), staging_dir) if fast_msi
```

Comment:
We should add some comments mentioning how this `*.dll` is built and where the source resides (I know you mentioned https://github.com/chef/fastmsi-custom-action on another PR).